src/src/config.h

```diff
@@ -0,0 +1,40 @@
+#pragma once 
+
+#include <Arduino.h>
+#include "targets.h"
+#include "elrs_eeprom.h"
+
+#define CONFIG_VERSION          1
+#define RESERVED_EEPROM_SIZE    512
```

Comment:
Already defined in elrs_eeprom.h.  Should it still be here?

Reply:
No, it should not.